operators/api/v1alpha2/template_types.go

```diff
@@ -122,9 +122,6 @@ type Environment struct {
 // EnvironmentResources is the specification of the amount of resources
 // (i.e. CPU, RAM, ...) assigned to a certain environment.
 type EnvironmentResources struct {
-	// +kubebuilder:validation:Minimum:=1
-	// +kubebuilder:validation:Maximum:=8
```

Comment:
Could make sense to keep at least the check that the number is non-negative (or leave the minimum set to one), to prevent strange failures.